README.kotlin.md

```diff
@@ -0,0 +1,18 @@
+# Iroh Kotlin
+
+## Requirements
+
+### CLI with `kotlinc`
+
+- [`kotlinc`](https://kotlinlang.org/docs/command-line.html)
+- [`ktlint`](https://github.com/pinterest/ktlint)
+- [`jna`](https://github.com/java-native-access/jna)
+
+### Android Development
+
+- Install the NDK you want to use, using Android Studio.
+- Configure rust to use it (TODO: add example)
```

Comment:
remove todo and add issue, there is something unsettling about having a "todo" in a readme that would make me not want to try it out

Reply:
fixed